src/pages/index.js

```diff
@@ -154,7 +154,7 @@ class Home extends Component {
                     </CtaItem>
                     <CtaItem>
                       <ButtonLink to="/tutorial/tutorial.html" type="secondary">
-                        Haz el tutorial
+                        Toma el tutorial
```

Comment:
Hey @icarlossz, these two are pretty much the same, let's leave them as they are for now.

```suggestion
                        Haz el tutorial
```

Reply:
Done